### ISSUE TITLE
Orderby issue

### ISSUE THREAD

**Issue by zpaulovics** [Soontao/odata-v4-server#97]
Hi Theo,

Request:
```
http://localhost:3000/v4/Proposals?%24orderby=name
```
Respond:
```
 {"error":{"code":500,"message":"ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`proposal`.`name` ASC' at line 1"}}
```

Here is the MySql log:
```
query: START TRANSACTION
query: SELECT * FROM `proposal`  ORDERBY `proposal`.`name` ASC;
query failed: SELECT * FROM `proposal`  ORDERBY `proposal`.`name` ASC;
error: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`proposal`.`name` ASC' at line 1
```

When I do not use the $orderby this is the MySql log:
```
query: START TRANSACTION
query: SELECT `Proposal`.`id` AS `Proposal_id`, `Proposal`.`name` AS `Proposal_name`, `Proposal`.`voltageLevel` AS `Proposal_voltageLevel`, `Proposal`.`consumerProfile` AS `Proposal_consumerProfile`, `Proposal`.`energyFee` AS `Proposal_energyFee`, `Proposal`.`funds` AS `Proposal_funds`, `Proposal`.`nogaranty` AS `Proposal_nogaranty`, `Proposal`.`garanty` AS `Proposal_garanty`, `Proposal`.`discounts` AS `Proposal_discounts`, `Proposal`.`green` AS `Proposal_green`, `Proposal`.`domastic` AS `Proposal_domastic`, `Proposal`.`loyalty` AS `Proposal_loyalty`, `Proposal`.`online` AS `Proposal_online`, `Proposal`.`yearly` AS `Proposal_yearly`, `Proposal`.`co2` AS `Proposal_co2`, `Proposal`.`startDate` AS `Proposal_startDate`, `Proposal`.`endDate` AS `Proposal_endDate`, `Proposal`.`isPublic` AS `Proposal_isPublic`, `Proposal`.`providerId` AS `Proposal_providerId` FROM `proposal` `Proposal`
query: COMMIT
```

Regads, Zoltan

**Comment by Soontao** [comment]
confirmed, its a SQL issue.

**Comment by Soontao** [comment]
New test cases added for `$orderby`

https://github.com/Soontao/odata-v4-server/blob/29f92a5568546b436336c4a08aa8db9e1e11ac83/test/type/server_query.spec.ts#L32-L84

**Comment by Soontao** [comment]
fixed after version `0.2.40`
